### DISCUSSION TITLE
Timeout option if no result is available within a certain time

### DISCUSSION THREAD

**Discussion by a-chatterjee** [convictional/trigger-workflow-and-wait#28] [Q&A]
Hi,

Are there any ways to pass a timeout value after which, it'd fail the task if no response is received from the triggered repository?

**Comment by duhow** [comment]
GitHub Actions already provides a `timeout-minutes` argument.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes
